packages/styles/scss/patterns/blocks/listsection/_listsection.scss

```diff
@@ -76,6 +76,17 @@
       }
     }
 
+    .#{$prefix}--content-group__title {
+      padding-left: 0;
+    }
+
+    .#{$prefix}--content-group {
+      padding-bottom: $carbon--layout-03;
+      &:last-of-type {
+        padding-bottom: 0;
+      }
+    }
+
```

Comment:
Adding @kennylam here, as he's working on the content block at the moment which includes refactoring listsection.